src/json/util.cc

```diff
@@ -62,6 +62,7 @@ const char *jsonutil_code_to_message(JsonUtilCode code) {
         case JSONUTIL_INVALID_RDB_FORMAT:
             return "ERROR Invalid value in RDB format";
         case JSONUTIL_DOLLAR_CANNOT_APPLY_TO_NON_ROOT: return "SYNTAXERR Dollar sign cannot apply to non-root element";
+        case JSONUTIL_MEMORY_ERROR: return "OOM Memory allocation failed";
```

Comment:
Why do we need this OOM error? json.mset is already marked as `deny-oom`. The valkey core should be able to handle the OOM failure.